src/endpoints/apollo.ts

```diff
@@ -0,0 +1,147 @@
+//#region > Imports
+//#PACKAGE "apollo-client"
+//## npm install "apollo-client"@2.6.8
+// Contains the client for graphql handling
+import { ApolloClient } from "apollo-client";
+//#PACKAGE "apollo-link-http"
+//## npm install "apollo-link-http"@1.5.16
+// Contains the link for the apollo client
+import { HttpLink } from "apollo-link-http";
+//#PACKAGE "apollo-cache-inmemory"
+//## npm install "apollo-cache-inmemory"@1.6.5
+// Contains cache handling for apollo
+import {
+  InMemoryCache,
+  IntrospectionFragmentMatcher,
+  NormalizedCacheObject
```

Comment:
Missing comma?
```suggestion
  NormalizedCacheObject,
```

Reply:
I just saw that there are more styling issues related to commas. I'll fix that with prettier.

Reply:
I just realised that those issues occurred because of different prettier settings of @pinterid.
Therefore @snek-at/frontend should check be aware of this issue, check their settings and/or find a way to provide prettier project settings.

Reply:
You can set prettier settings in `package.json`. This issue also caused the indentation issue we saw some weeks ago.

Reply:
There are multiple ways to define prettier settings. I would suggest to create a `. prettierrc` to keep it separated.

@Aichnerc Can you create a setting file with all prettier settings you are using, then we will implement this in all our projects.

The problem I see is that every user could have an off-standard configuration file. Therefore settings which are not overwritten by `.prettierrc` will affect the code.

Does @snek-at/frontend have a idea how to fix this?

Reply:
Bruh.